lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3376,6 +3399,29 @@ namespace {
                                 std::nullopt);
     }
 
+    bool isNormalCxxMemberMethod(const clang::CXXMethodDecl *method) {
+      // Check if the method is a constructor or destructor
+      if (llvm::dyn_cast<clang::CXXConstructorDecl>(method) ||
+          llvm::dyn_cast<clang::CXXDestructorDecl>(method)) {
+        return false;
+      }
+
+      // Exclude compiler-generated, special, and trivial functions
```

Comment:
I think we should be able to annotate some of the examples below. E.g., I do not see why overloaded operators would not work. In these cases I think it might be more valuable to add a test rather than an exclusion. 

Also, I doubt that we would ever import deleted functions into Swift since those cannot be called, since they are deleted.

Reply:
Moving this conversation into a different patch: https://github.com/swiftlang/swift/pull/77397